src/test/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoaderTest.java

```diff
@@ -122,4 +125,26 @@ public void shouldGetEpimsIdForGivenPostcode() {
         String result = venueDataLoader.getActiveVenueDetailsMapByPostcode().get("MK9 2AJ").getEpimsId();
         assertEquals("815997", result);
     }
+
+    @Test
+    public void testActiveVenueEpimsIdsMapByRpcReturnsVenues() {
+        List<String> result = venueDataLoader.getActiveVenueEpimsIdsMapByRpc().get("SSCS Leeds");
+
+        assertFalse(result.isEmpty());
+        assertEquals(20, result.size());
+        assertEquals(Arrays.asList("517400", "449358", "563156",
+                "45900", "744412", "572158",
+                "288691", "562808", "720624",
+                "427519", "366796", "999974",
+                "107581", "197852", "495952",
+                "852649", "491107", "195520",
+                "641199", "574546"), result);
+    }
+
+    @Test
+    public void testActiveVenueEpimsIdsMapByRpcReturnsNullOnErroneousInput() {
+        List<String> result = venueDataLoader.getActiveVenueEpimsIdsMapByRpc().get("SSCS Newcastle");
+
+        assertNull(result);
```

Comment:
Should return an empty list instead of null

Reply:
I'll amend the getActiveVenueEpimsIdsMapByRpc() to return empty list if no key found / null.

Reply:
Looking at this again, the service test will return empty list, the loader will return null as it's using get on a map where key doesn't exist. Think that should be fine in lieu of the other maps / tests for the loader.

Reply:
Wouldn't it be safer for the service to return an empty list or an optional?